docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -98,40 +98,53 @@ export S3_ENDPOINT_URL='https://<accountid>.r2.cloudflarestorage.com'
 For [MosaicML platform](https://www.mosaicml.com/cloud) users, follow the steps mentioned in the [Google Cloud Storage](https://mcli.docs.mosaicml.com/en/latest/secrets/gcp.html) MCLI doc on how to configure the cloud provider credentials.
```

Comment:
Can you also update the MosaicML GCS doc link from [this](https://mcli.docs.mosaicml.com/en/latest/secrets/gcp.html) to [this](https://docs.mosaicml.com/projects/mcli/en/latest/resources/secrets/gcp.html)

Reply:
Updated in 8140808a5d0ce065f6936f65b94be00b4cee12b8